src/components/FeeCollapse/index.tsx

```diff
@@ -4,30 +4,30 @@ import { roundDownToTwoDecimals } from '../../helpers/parseNumbers';
 import { OutputTokenDetails } from '../../constants/tokenConfig';
 import { useEventsContext } from '../../contexts/events';
 
-export function calculateTotalReceive(toAmount: string, outputToken: OutputTokenDetails): string {
+export function calculateTotalReceive(toAmount: Big, outputToken: OutputTokenDetails): string {
```

Comment:
Made more sense to use `Big` directly in the argument. Before we changed a `Big` to a `string` to call this function, only to change it back into a `Big` inside.